plugin/PySrc/mock_pyglet.py

```diff
@@ -0,0 +1,45 @@
+import sys
```

Comment:
You can move this into the `mock_turtle.py` file. I'm trying to limit the number of files needed to install the Emacs version.

Reply:
Ok this has now been appended to that file.